doc/releases/release-notes-3.7.rst

```diff
@@ -861,7 +861,7 @@ Drivers and Sensors
 
   * Added GNSS device driver API test suite.
   * Added support for the u-blox UBX protocol.
-  * Added device driver for the u-blox M10 GNSS modem (:dtcompatible:`u-blox,m10`).
+  * Added device driver for the u-blox M8 GNSS modem (:dtcompatible:`u-blox,m8`).
```

Comment:
It's nice to fix the 3.7 release notes, but there the DT compatible (and driver name) will still be M10. Unless this is backported?

Reply:
I'm not sure that this is worthy of backporting, it is not a bug or a security vulnerability, just a name correction.
The choice is either leave the old release notes alone, and the link will be dead, or I can update the old notes and have the name be inconsistent.
Neither is ideal.

Reply:
Yeah...